conans/client/command.py

```diff
@@ -561,27 +582,83 @@ def imports(self, *args):
+                                          " but a folder containing a conanfile.py or conanfile.txt"
+                                          " file.")
         except ConanException:
             pass
 
-        return self._conan.imports(args.path, args.dest, args.file, args.build_folder)
+        return self._conan.imports(args.path, args.dest, args.file, args.info_folder)
+
+    def export_pkg(self, *args):
+        """Exports the specified recipe and then creates a package binary from given precompiled
+           artifacts in user folder, skipping the build() method of the recipe, by calling the
+           package() method to extract the artifacts from the local "--source_folder" and
+           "--build_folder".
+        """
+        parser = argparse.ArgumentParser(description=self.export_pkg.__doc__,
+                                         prog="conan export-pkg .")
+        parser.add_argument("path", help='path to a recipe (conanfile.py). e.j: "." ')
+        parser.add_argument("reference", help='user/channel, or a full package reference'
+                                              ' (Pkg/version@user/channel), if name and version '
+                                              ' are not declared in the recipe (conanfile.py)')
+        parser.add_argument("--source-folder", "--source_folder", "-sf",
+                            help="local folder containing the sources. Defaulted to the directory "
+                                 "of the conanfile. A relative path can also be specified "
+                                 "(relative to the current directory)")
+        parser.add_argument("--build-folder", "--build_folder", "-bf",
+                            help="build folder, working directory of the build process. Defaulted "
+                                 "to the current directory. A relative path can also be specified "
+                                 "(relative to the current directory)")
+        parser.add_argument("--profile", "-pr",
+                            help='Profile for this package')
+        parser.add_argument("--options", "-o",
+                            help='Options for this package. e.g., -o with_qt=true',
+                            nargs=1, action=Extender)
+        parser.add_argument("--settings", "-s",
+                            help='Settings for this package e.g., -s compiler=gcc',
+                            nargs=1, action=Extender)
+        parser.add_argument("--env", "-e",
+                            help='Environment variables that will be set during the package build, '
+                                 '-e CXX=/usr/bin/clang++',
+                            nargs=1, action=Extender)
+        parser.add_argument('-f', '--force', default=False,
+                            action='store_true', help='Overwrite existing package if existing')
+        parser.add_argument('--no-export', '-ne', help='Do not export the recipe', default=False,
```

Comment:
I prefer to make the parameter "path" mandatory but specify if you don't want to export again the recipe. I think it's more clear to always read the local one. And more uniform with all the commands (beta)

Reply:
Sorry, but "reads a bit weird" (also not agree) is better than "I'm changing the path to force the command to not find the conanfile so it won't export it again"<=crazy. Any other purpose?